Users/Tutorial.rst

```diff
@@ -273,11 +283,13 @@ coala would now fix all spacing issues and without bothering you again.
 Setting Inheritance
 -------------------
 
-All settings in the default section are implicitly inherited to all
-other sections (if they do not override their values). We can use that
-to save a few lines!
+Let us focus on the ``[WARNING]`` part of the above output. It says
+that all settings in the ``cli`` section are implicitly inherited to all
+other sections (if they do not override their values). It also advises
+us to change the name of that section to avoid unexpected behavior.
```

Comment:
I would dedicate a note to the `cli` section mentioning and explaining its existence. But it shouldn't be part of the main stream of the doc.